ci/coverity/model_file.c

```diff
@@ -0,0 +1,88 @@
+#define LargestIntegralType unsigned long long
+
+void _assert_true(const LargestIntegralType result,
+                  const char* const expression,
+                  const char * const file, const int line)
+{
+      __coverity_panic__();
+}
+
+void _assert_int_equal(
+    const LargestIntegralType a, const LargestIntegralType b,
+    const char * const file, const int line)
+{
+      __coverity_panic__();
+}
+
+void _assert_int_not_equal(
+    const LargestIntegralType a, const LargestIntegralType b,
+    const char * const file, const int line)
+{
+      __coverity_panic__();
+}
+
+void _assert_return_code(const LargestIntegralType result,
+                         size_t rlen,
+                         const LargestIntegralType error,
+                         const char * const expression,
+                         const char * const file,
+                         const int line)
+{
+      __coverity_panic__();
+}
+
+void _assert_string_equal(const char * const a, const char * const b,
+                          const char * const file, const int line)
+{
+      __coverity_panic__();
+}
+
+void _assert_string_not_equal(const char * const a, const char * const b,
+                              const char *file, const int line)
+{
+      __coverity_panic__();
+}
+
+void _assert_memory_equal(const void * const a, const void * const b,
+                          const size_t size, const char* const file,
+                          const int line)
+{
+      __coverity_panic__();
+}
+
+void _assert_memory_not_equal(const void * const a, const void * const b,
+                              const size_t size, const char* const file,
```

Comment:
(style)  code indent should use tabs where possible
(style)  "foo* const bar" should be "foo * const bar"